packages/truffle/scripts/test.sh

```diff
@@ -3,7 +3,7 @@
 set -o errexit
 
 if [ "$GETH" == true ]; then
-  mocha --timeout 50000 --grep @ganache --invert --colors $@
+  mocha --timeout 50000 --grep '@ganache|@standalone' --invert --colors $@
```

Comment:
This isn't just `@geth`?

Reply:
i guess it could be, but I figure we want to make sure anything generalized having to do with migrations works with geth on this job.